src/migrations/20220610171512-update-unique-keys-LeaderboardResult_table.ts

```diff
@@ -0,0 +1,42 @@
+import type { QueryInterface, Sequelize } from 'sequelize';
+
+interface SequelizeContext {
+  context: {
+    queryInterface: QueryInterface;
+    Sequelize: Sequelize;
+  };
+}
+
+module.exports = {
+  async up({ context: { queryInterface } }: SequelizeContext) {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addIndex('LeaderboardResults', {
+        name: 'index_user_leaderboard',
+        fields: ['_userId', '_leaderboardEntryId'],
+        unique: true,
+        transaction,
+      });
+
+      await queryInterface.addIndex('LeaderboardResultsMeta', {
+        name: 'index_attribute_leaderboardresult',
+        fields: ['attribute', '_leaderboardResultsId'],
+        unique: true,
+        transaction,
+      });
+    });
+  },
```

Comment:
added indexes so that one user will only be allowed to have a single leaderboard result per leaderboard
also added a index so that one unqiue attribute can be saved per leaderboard result